WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/StatsPage.kt

```diff
@@ -160,15 +163,14 @@ class StatsPage {
         return this
     }
 
-    private fun scrollToCard(cardHeader: String) {
-        val card = Espresso.onView(
-            Matchers.allOf(
-                ViewMatchers.isDescendantOfA(visibleCoordinatorLayout),
-                ViewMatchers.withId(R.id.stats_block_list),
-                ViewMatchers.hasDescendant(ViewMatchers.withText(cardHeader))
-            )
+    private fun scrollToCard(viewholderPosition: Int, section: StatsListViewModel.StatsSection) {
+        WPSupportUtils.idleFor(2000)
+        Espresso.onView(Matchers.allOf(
+            ViewMatchers.withTagValue(Matchers.`is`(section.name))
+        )).perform(
+            RecyclerViewActions.scrollToPosition<ViewHolder>(viewholderPosition) // This works
```

Comment:
Is the comment of `// This works` for your development purpose and okay to remove?